lib/storage-handler.js

```diff
@@ -221,11 +223,13 @@ exports.upload = function(provider, req, res, options, cb) {
 
     if (err) {
       console.error(err);
+      debug('Form parsing error: %o', err);
```

Comment:
I think this debug statement is not needed since we log the error to stderr and send it to the callback.